config/application.rb

```diff
@@ -57,6 +57,8 @@ class Application < Rails::Application
     config.i18n.available_locales = Locale.available_locales.sort_by(&:swapcase).map &:to_sym
     config.i18n.default_locale = :en
 
+    config.action_view.sanitized_allowed_tags = Rails::Html::Sanitizer.white_list_sanitizer.allowed_tags + ['strike', 'u']
+
```

Comment:
@julianguyen I have whitelisted these two tags globally, let me know If I need to whitelist only in required place.